### DISCUSSION TITLE
Pro and non Pro

### DISCUSSION THREAD

**Discussion by mycanaletto** [rustdesk/rustdesk#5819] [General]
### Bug Description

I've installed a Pro server, purchased a license and it works pretty well.

The problem :

When it comes to connecting to the clients I manage, no problem. However, if I need to connect to an isolated client to which I don't want to give the server key and info, the connection is impossible...

### How to Reproduce

Easy

### Expected Behavior

N/A

### Operating system(s) on local side and remote side

W10

### RustDesk Version(s) on local side and remote side

1.2.2

### Screenshots

N/A

**Comment by dinger1986** [comment]
You can rename the exe as explained here https://rustdesk.com/docs/en/self-host/client-configuration/

**Comment by mycanaletto** [comment]
I don't want to give the key to my customer, just have him run a standard rustdesk and give me the password....

**Comment by rustdesk** [comment]
`Being controlled` side does not require `key`, you can give `host` only. But our `rename` and `--config` require `host` and `key` both, we may have to do some code change for your case.

**Reply by pdifeo** [reply]
Interesting feature if it is implemented.

In any case, I configured a client with only the server name and no key.

I tried to connect to that client but I can't. This is the screenshot

![image](https://github.com/rustdesk/rustdesk/assets/8618744/008c1fa8-16f4-4a99-a733-5f41095cfcc4)

**Reply by rustdesk** [reply]
I tested, it was ok for me. @dinger1986 can you also test?

**Reply by dinger1986** [reply]
same for me, as soon as I remove the key it doesnt connect, tried with 1.2.2 and nightly builds as well

**Reply by rustdesk** [reply]
`Being controlled` side?

**Reply by rustdesk** [reply]
Interesting, working fine on my side.

![image](https://github.com/rustdesk/rustdesk/assets/71636191/12f5f4f6-b033-46dc-b25c-e53e1cf5796b)

**Comment by dinger1986** [comment]
![accesswithoutkey](https://github.com/rustdesk/rustdesk/assets/7244447/f931d37f-79e9-449c-8729-b70bf26162db)

Did a gif

**Reply by dinger1986** [reply]
was blank without the = the first time but not when I recorded it

**Comment by mycanaletto** [comment]
Whether it works or not, we're not going to change the configuration every time we need to connect to an isolated client....

What's more, you need to be an administrator of the workstation, which is not the case for support staff.

In the developer's concept, you have to imagine that there are many of us who want to replace TeamViewer, but that there are a minimum number of things we need to be able to do with Rustdesk....

**Reply by rustdesk** [reply]
please subscribe to this issue https://github.com/rustdesk/rustdesk-server-pro/issues/55
